gnocchi/tests/functional/gabbits/metric-list.yaml

```diff
@@ -89,7 +89,7 @@ tests:
     - name: list metrics
       GET: /v1/metric
       response_json_paths:
-          $.`len`: 2
+          $.`len`: 4
```

Comment:
tbh, i don't really understand what the goal is. i don't think this makes sense that user foobar can list metrics from user foobaz and user jd.

we should probably add more tests to show what you're trying to achieve.

Reply:
I hoped the commit message was pretty explicit.

Reply:
uh... so do we have the equivalent keystone tests to ensure that path still does what it should?

Reply:
Yes, it's in `test_rest`, that's the only place we test ACL.